docs/standards/CX-0135-CompanyCertificateManagement/CX-0135-CompanyCertificateManagement.md

```diff
@@ -172,19 +279,30 @@ Certificate is pushed by the provider to the consumer. The enclosed Bpns can be
   }
 }
 ```
+The ´senderFeedbackUrl´ specifies, where the Certificate Distributor expects the feedback of the Certificate Receiver.
+The expected value **MUST** be a concrete path to the version 1 dataspace protocol endpoint (as specified in the example above),
+where a data offer for an asset of type cx-taxo:CCMAPI **MUST** be available for the Certificate Receiver.
+
+>**Push header `senderFeedbackUrl` explanation**: 
+>This information is intended as a temporary solution to support the unique identification of multiple feedback endpoints across multiple EDCs belonging to one legal entity. 
+>The typical way to implement such differentiation in the Catena-X data space would be to provide additional, distinguishing attributes to the EDC assets to enable an automated search mechanism via the EDC discovery service and EDC catalogs. 
+>Since the current changes are implemented as a non-breaking standard patch, the senderFeedbackUrl remains an intermediate solution. 
+>A future change is required in that regard, especially when considering the deprecation of the v1 DSP endpoint in favor of an upcoming EDC `.well-known` endpoint that supports multiple DSP versions.
 
-**Company Certificate Status**
+##### 2.1.1.3 Company Certificate Feedback
```

Comment:
Renaming this only here without renaming it on other occasions (e.g. in the `context`) is just introducing more complexity and confusion.
If you plan to rename it, please make sure to do that everywhere.

Reply:
Ok, i understand.
All in all a good change but please consider also changing other occasions (like the `context`) to avoid confusion.